README.md

```diff
@@ -1 +1,70 @@
-Interface mockup
\ No newline at end of file
+# Microscopium User Interface
+
+Prerequisites
+-------------
+
+* Node.js > v0.10.35
+
+Install Instructions
+--------------------
+
+```
+$ npm install
+$ bower install
+```
+
+Run dev version locally with
+```
+grunt
+```
+
+MongoDB Schema
+--------------
+
+Screens saved in document with format:
+
+```
+var Screen = {
+    _id: String, // short screen name < 8 char
+    screen_name: String, // full screen name
+    screen_desc: String,
+    number_samples : Number
+    features : Array // Array of features used in the screen
```

Comment:
Array of String? If this is indeed the feature names, please specify in the comment string.